lib/value.ml

```diff
@@ -343,6 +343,21 @@ and polymorphic_function names expr =
     The loop goes over the possible key-value pairs in the list and accumulates
     the possible values in a list. Once complete whatever the last value was is
     used in the construction of the record. *)
+
+(* let get_attribute attr ld ~f =
```

Comment:
If you could remove these commented out pieces of code when you're ready that would be good.